module/VuFind/src/VuFind/ILS/Logic/Holds.php

```diff
@@ -346,7 +346,7 @@ protected function generateHoldings($result, $type, $holdConfig)
                     $groupKey = $this->getHoldingsGroupKey($copy);
                     $holdings[$groupKey][] = $copy;
                     // Are any copies available?
-                    if ($copy['availability'] == true) {
+                    if ($copy['availability']->isAvailable()) {
```

Comment:
Is it worth putting an `__invoke` method in the class that wraps around `isAvailable`, so if somebody has an outdated template, it's more likely to behave as expected?

Reply:
Isn't `$copy['availability']` is the AvailabiltyStatus object and if we want to call `__invoke` we would have to add brackets? So I'm not sure if we would win anything by this. It even might be less obvious to see the difference.

Reply:
You are, of course, correct. I think my brain must have switched into some other programming language when I was thinking about this. ;-)